packages/discovery-react-components/src/components/TooltipHighlight/TooltipHighlight.tsx

```diff
@@ -162,6 +159,28 @@ export function calcToolTipContent(
   return tooltipContent;
 }
 
+function calcOneTooltipRow(
+  tableContent: any[],
+  facetInfoMap: FacetInfoMap,
+  facetId: string,
+  enrichValue: string
+) {
+  // Account for invalid facetId, fail gracefully
+  let facetInfo = facetInfoMap[facetId] || {};
+  let enrichColor = facetInfo.color || '';
+  let enrichFacetDisplayname = facetInfo.displayName || '';
+  if (enrichFacetDisplayname.localeCompare(enrichValue, undefined, { sensitivity: 'base' }) === 0) {
+    // This case applies to keyowrds
+    enrichFacetDisplayname = KEYWORDS_CATEGORY;
+  }
+  // Will have multiple entries after overlapping is implemented
```

Comment:
Update this comment, since overlapping is now implemented. Maybe:
```suggestion
  // Will have multiple entries if overlapping
```

Reply:
done